src/overlays/actors/ovl_Set_Manager/ac_set_manager.c

```diff
@@ -2,52 +2,200 @@
 #include "m_lib.h"
 #include "m_actor_dlftbls.h"
 #include "m_object.h"
+#include "m_player_lib.h"
+#include "m_field_info.h"
+#include "ovlmgr.h"
+#include "attributes.h"
 #include "overlays/gamestates/ovl_play/m_play.h"
+#include "overlays/actors/player_actor/m_player.h"
+
+// Fish and Insect overlay managers
+#include "overlays/managers/ovl__00821B40/ovl__00821B40.h"
+#include "overlays/managers/ovl__008253C0/ovl__008253C0.h"
 
 void aSetMgr_ct(Actor* thisx, Game_Play* game_play);
 void aSetMgr_dt(Actor* thisx, Game_Play* game_play);
-void aSetMgr_move(Actor* thisx, Game_Play* game_play);
+void aSetMgr_move(Actor* thisx, Game_Play* play);
 
-#if 0
-ActorProfile Set_Manager_Profile = {
-    /* */ ACTOR_SET_MANAGER,
-    /* */ ACTOR_PART_7,
-    /* */ ACTOR_FLAG_10 | ACTOR_FLAG_20 | ACTOR_FLAG_20000000,
-    /* */ 0x0000,
-    /* */ GAMEPLAY_KEEP,
-    /* */ sizeof(Set_Manager),
-    /* */ aSetMgr_ct,
-    /* */ aSetMgr_dt,
-    /* */ aSetMgr_move,
-    /* */ (void*)none_proc1,
-    /* */ NULL,
+s32 aSetMgr_check_player_wade_end(void);
+s32 aSetMgr_check_player_wade_start(void);
+
+SetMgrOvlInfo proc_table[SETMGR_OVERLAY_MAX] = {
+    SETMGR_OVERLAY(ovl__00821B40, func_8092A030_jp, func_8092AF0C_jp), // Fish
+    SETMGR_OVERLAY(ovl__008253C0, func_8092DBC0_jp, func_8092ECAC_jp), // Insect
```

Comment:
I guess we don't have any names for those :c

Reply:
We do have them, but I thought it would be better to name them after I work on these, I can rename them already though

Reply:
It's weird, because in ac they do have the prefix of actors in the namefile but they're not actors lol, unsure why they did that

Reply:
I think it would be better to name them now